### DISCUSSION TITLE
close menue

### DISCUSSION THREAD

**Discussion by Sim0nW0lf** [kimai/kimai#3852] [General]
Will it be possible in Version 2 to close/minimize the side menu like we were able to in the last version?
If not might there be a way to do so?

![Screenshot_20230217_150742_edit_155757440493420.jpg](https://user-images.githubusercontent.com/31454341/219679080-8a37f920-51bc-437d-b616-6f441ad2f697.jpg)

**Comment by kevinpapst** [comment]
I don't have plans to add such a feature. 

I'd say: regular users do not need that feature and all other users, that do reporting and such, should own a large enough screen.

What is the true use-case that you are referencing?
